addons/l10n_in/models/chart_template.py

```diff
@@ -13,10 +13,15 @@ def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
             for journal in res:
                 if journal.get('type') in ('sale','purchase'):
                     journal['l10n_in_gstin_partner_id'] = company.partner_id.id
-                if journal['code'] == 'INV':
-                    journal['name'] = _('Tax Invoices')
         return res
 
+    def _load(self, sale_tax_rate, purchase_tax_rate, company):
+        res = super(AccountChartTemplate, self)._load(sale_tax_rate, purchase_tax_rate, company)
+        if self == self.env.ref("l10n_in.indian_chart_template_standard"):
```

Comment:
why not: if the company.account_fiscal....  == "IN"

Reply:
this deserves more details: in comment or in the commit message itself

Reply:
> why not: if the company.account_fiscal.... == "IN"

the country is not required in the company so if I do not set the county in the company and load the chart template then it's not work.

and what problem with this?

Reply:
yes, we need to set when install the CoA